### DISCUSSION TITLE
100ms mode

### DISCUSSION THREAD

**Discussion by tphummel** [BattlesnakeOfficial/feedback#117] [General]
Would it be worthwhile to have a mode that is the opposite of galaxy mind? 250ms or 100ms response times required?

**Comment by bvanvugt** [comment]
Could be fun! 100 might be too tight for most hosting setups, but 200/250 is reasonable. What's a good name for it?

**Reply by tphummel** [reply]
Mini-mind
Quick-mind?

**Reply by chris-bsnake** [reply]
We have now added the new Global Royale (Speed Snake) arena with a 200ms response time.

[https://play.battlesnake.com/arena/global-royale-speed-snake/](https://play.battlesnake.com/arena/global-royale-speed-snake/)

**Reply by tphummel** [reply]
This is rad. I'm in.
